### PR TITLE
Added code for sending mail after user creation[Single and Bulk Import]

### PR DESCRIPTION
**Pull request by Krishna92Soni**

### CODE REVIEW

File: APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs [hunk 1]
```diff
@@ -172,6 +176,33 @@ public async Task<JsonResult> Create(CreateUserVM user)
                     var userId = connection.Query<int>(procedure, parameters, commandType: CommandType.StoredProcedure).FirstOrDefault();
                     if (userId > 0)
                     {
+                        #region Send Mail
+                        byte[] byteForEmail = Encoding.ASCII.GetBytes(user.Email);
+                        string encryptedEmail = Convert.ToBase64String(byteForEmail);
+                        UriBuilder builder = new();
+                        builder.Scheme = "http";
+                        builder.Host = "localhost";
+                        builder.Port = 4200;
```
Comment (mittalvyas1) [inline APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs:185]: Can't use static values

**Comment by mittalvyas1** [comment]
Please do self review before creating PR.

File: APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs (hunk 1 above)
Comment (Krishna92Soni) [reply APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs]: resolved

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

File: APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs [hunk 2]
```diff
@@ -172,6 +176,32 @@ public async Task<JsonResult> Create(CreateUserVM user)
                     var userId = connection.Query<int>(procedure, parameters, commandType: CommandType.StoredProcedure).FirstOrDefault();
                     if (userId > 0)
                     {
+                        #region Send Mail
```
Comment (mittalvyas1) [inline APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs:179]: Transfer Email functionality in common class

**Review by mittalvyas1 — APPROVED** [review]

**Pull request merged**